app/models/katello/docker_manifest_list.rb

```diff
@@ -25,7 +25,7 @@ def update_from_json(json)
       update_attributes(:schema_version => json[:schema_version],
                         :digest => json[:digest],
                         :downloaded => json[:downloaded],
-                        :docker_manifests => ::Katello::DockerManifest.where(:digest => json[:manifests])
+                        :docker_manifests => ::Katello::DockerManifest.where(:digest => json[:manifests].pluck(:digest))
```

Comment:
If `json` is actual JSON, will pluck work here? I thought it was an AR method only

Reply:
@johnpmitsch - It works, which surprised me, and is very readable so I went that way.